doc/changes/changes_1.7.0.md

```diff
@@ -19,5 +20,6 @@ If you need further versions, please open an issue.
 
 ## Changes
 
+* #301: Added commanline option `--docker-access-method`
```

Comment:
Can we find a shorter but as precise term for the option? E.g. `--access-scheme`

Reply:
Actually, we probably should think in general over a better name, because we only use this for accessing the file system of the DB.

Reply:
Which term should I apply for the current PR?
`--access-scheme` sound a little too general to me. It's currently only about Docker Containers and in fact applies only to the Docker Container containing the Exasol database. So to be precise the name should rather be longer than shorter 🙂 

On the other hand I think this option will not be used too often: 
* As soon as the new flavor is released officially we can use a specific version number to identify the flavor and can remove the option. 
* As long as the new flavor is **not** released officially we rarely will need this option, anyway.

So for the moment `--docker-access-method` is still my favorite. What do you think?

Reply:
Proposal `--db-os-access`.